Makefile

```diff
@@ -633,6 +633,17 @@ endif
 	git commit -m "Adding $(MLRUN_VERSION) tag contents" --allow-empty; \
 	git push origin $$BRANCH_NAME
 
+.PHONY: api-backward-compatibility
+api-backward-compatibility: ## Check for api backward compatibility breakage
+ifndef BASE_PATH
+	$(error BASE_PATH is undefined)
+endif
+	mkdir base && mkdir head ;\
+	export OPENAPI_JSON_TARGET_PATH=$(PWD)/base &&\
+	python -m pytest -v $(BASE_PATH)/tests/api/api/test_docs.py::test_save_openapi_json && \
+	export OPENAPI_JSON_TARGET_PATH=$(PWD)/head && \
+	python -m pytest -v $(PWD)/tests/api/api/test_docs.py::test_save_openapi_json && \
+	docker run --rm -t -v $(PWD):/specs:ro openapitools/openapi-diff:2.0.1 /specs/base/openapi.json /specs/head/openapi.json --fail-on-incompatible > d.diff
```

Comment:
I see you're writing the output of the docker command to a file, IIUC it means it won't be written to the stdout, meaning a CI that fails (or succeeds) won't have any logs ?